data/campaigns/Heir_To_The_Throne/scenarios/19c_Cliffs_of_Thoria.cfg

```diff
@@ -899,6 +899,7 @@ Soooo... It is you who sent your subordinates to attack us. Now when we’ve des
 
     [event]
         name=moveto
+        first_time_only=no
```

Comment:
This still prevents undoing of that move. Thus the better fix would be to either use a [filter_condition] check instead of the [if], or to turn the event into a simple nested event (with the waterfall event as its parent) so that it doesn't need to check the variable in the first place.

The commit message is also pretty unclearly formatted.